public/js/mstream.js

```diff
@@ -483,7 +544,7 @@ $(document).ready(function () {
           newArray.push(fileExplorerArray[i]);
         }
 
-        senddir(newArray);
+        senddir(newArray, restorePreviousExplorerListState);
```

Comment:
the passed function is executed when api call completes, that's why I cant just call it right here